src/frontend/src/components/message.ts

```diff
@@ -0,0 +1,33 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+import { renderPage } from "../utils/lit-html";
+
+const showMessageTemplate = ({
+  message,
+  role,
+}: {
+  message: string;
+  role?: string;
+}): TemplateResult => {
+  return html`<h1
```

Comment:
not for this PR: should it be a h1 ?

Reply:
Good question; probably not. Added to todo list alongside your comment about making the css attributes into a class